docs/docs/en/development/development-environment-setup.md

```diff
@@ -152,7 +152,7 @@ There are three services that need to be started, including MasterServer, Worker
 Install frontend dependencies and run it
 
 ```shell
-cd dolphinscheduler-ui-next
+cd dolphinscheduler-ui
```

Comment:
should also change Chinese docs in https://dolphinscheduler.apache.org/zh-cn/development/development-environment-setup.html, BTW, should we also need to remove content in https://dolphinscheduler.apache.org/en-us/development/frontend-development.html because in out of date now

Reply:
Chinese file link is  https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/zh/development/development-environment-setup.md